### PR TITLE
feat: multiple connections selection for proof request

### PR DESCRIPTION
**Pull request by pranalidhanavade**
### What

- Support to select multiple selection of connections while requesting proof by connections.

### CODE REVIEW

File: src/components/Verification/ConnectionList.tsx [hunk 1]
```diff
@@ -63,6 +73,154 @@ const ConnectionList = (props: {
...
+			);
+		})();
+	}, [localOrgs]);
+
+
+	useEffect(() => {
+		let getData: NodeJS.Timeout;
+		updateLocalOrgs();
+		if (listAPIParameter?.search?.length >= 1) {
+			getData = setTimeout(() => {
```
Comment (prasadgkalamkar) [inline src/components/Verification/ConnectionList.tsx:210]: What is Data here? We can have relevant variable names.

File: src/components/Verification/ConnectionList.tsx [hunk 2]
```diff
@@ -79,61 +237,8 @@ const ConnectionList = (props: {
...
-									),
-								},
-							],
-						};
-					},
-				);
-
-				setConnectionList(connections);
+				
+				setConnectionList(data?.data?.data);
```
Comment (prasadgkalamkar) [inline src/components/Verification/ConnectionList.tsx:241]: can we change this data.data.data?

File: src/components/Verification/ConnectionList.tsx [hunk 3]
```diff
@@ -223,7 +301,7 @@ const ConnectionList = (props: {
 				onInputChange={searchInputChange}
 				refresh={refreshPage}
 				header={verification_header}
-				data={connectionList}
+				data={tableData}
```
Comment (prasadgkalamkar) [inline src/components/Verification/ConnectionList.tsx:304]: Which data table has? rename variable for the same.

**Review by prasadgkalamkar — CHANGES_REQUESTED** [review]

File: src/components/Verification/ConnectionList.tsx (hunk 1 above)
Comment (pranalidhanavade) [reply src/components/Verification/ConnectionList.tsx:210]: I will change name of variable accordingly

File: src/components/Verification/ConnectionList.tsx (hunk 2 above)
Comment (pranalidhanavade) [reply src/components/Verification/ConnectionList.tsx:241]: Yes

File: src/components/Verification/ConnectionList.tsx (hunk 3 above)
Comment (pranalidhanavade) [reply src/components/Verification/ConnectionList.tsx:304]: Yes.

**Review by prasadgkalamkar — APPROVED** [review]

**Pull request merged**